repository/migration/migrate_db_013_to_XXX_MariaDB_step_0.sql

```diff
@@ -308,4 +308,10 @@ begin
                  inner join user_role ur on tenant_user.id = ur.tenant_user_id
         where role = 'SYS_ADMIN';
     end if;
-end;
\ No newline at end of file
+end;
+
+create procedure add_workspace_id_and_populate()
```

Comment:
Shouldn't you be also calling the procedure? 
I also recommend commenting as per the other procedures (top of the script) if you can, even if this one is smaller.

Reply:
I absolutely should call it - that got somehow lost in the commits. Will re-add right away. 

Yeah I'll drop a small comment on there.

Reply:
Cheerios